static/third/bootstrap-typeahead/typeahead.js

```diff
@@ -447,6 +448,12 @@
       }, 150)
     }
 
+  , click_outside: function (e) {
```

Comment:
This could do with a better name -- I think we want something more like `element_clicked` -- since `click_outside` sounds like it's referring to clicks anywhere, not just on `$element`.

Reply:
Got it. Renamed to `element_click` (to maintain consistent present tense)